tests/pds/dataservice_test.go

```diff
@@ -745,6 +745,101 @@ var _ = Describe("{UpgradeDataServiceImage}", func() {
 	})
 })
 
+var _ = Describe("{DrainAndDecommissionNode}", func() {
+	JustBeforeEach(func() {
+		StartTorpedoTest("DrainAndDecommissionNode", "Deploys a data service, drains one selected node, decommissions that node", pdsLabels, 0)
+	})
+
+	It("Drain and Decommission a node", func() {
+		Step("Deploy, Validate, Drain Node, Cordon Node, Validate Data Service, Run Workload on Data Service, Uncordon Node", func() {
+			for _, ds := range params.DataServiceToTest {
+				isDeploymentsDeleted = false
+				deployment, _, _, err = DeployandValidateDataServices(ds, params.InfraToTest.Namespace, tenantID, projectID)
+				log.FailOnError(err, fmt.Sprintf("Error while deploying data services %s", ds.Name))
+				log.InfoD("Running Check Node DataService Func %v ", ds.Name)
+				nodes, err := pdslib.GetNodesOfSS(*deployment.ClusterResourceName, namespace)
+				log.FailOnError(err, "Cannot fetch nodes of the running Data Service")
+				var nodeName string
```

Comment:
can we move this declaration to top of the testcase

Reply:
Done